eng/templates/Azure.ResourceManager.Template/src/Azure.ResourceManager.Template.csproj

```diff
@@ -4,8 +4,6 @@
     <PackageId>Azure.ResourceManager.Template</PackageId>
     <Description>Azure Resource Manager client SDK for Azure resource provider ProviderFullName</Description>
     <PackageTags>azure;management;arm;resource manager;ProviderNameLowercase</PackageTags>
+    <NoWarn>$(NoWarn);CS1591;</NoWarn>
```

Comment:
This shouldn't be added we should have xml comments on all our publicly visible methods and properties.